template.tpl

```diff
@@ -441,6 +502,14 @@ var initData = data.advancedMatchingParams && data.advancedMatchingParams.length
 initData.integration = 'gtm';
 initData.useDecimalCurrencyValues = true;
 
+var dataProcessingOptions = data.dataProcessingParams && data.dataProcessingParams.length ? makeTableMap(data.dataProcessingParams, 'name', 'value') : null;
+if (dataProcessingOptions && dataProcessingOptions.mode && dataProcessingOptions.mode !== '') {
+  initData.dpm = dataProcessingOptions.mode;
```

Comment:
important to note the in pixelJS we can allow array, but here because of the text box we can allow string. LMK if we should also make it compatible with comma separated input and we convert it into an array before init-ing the pixel

Reply:
Or if any other input format is more suitable for this. IMO using a text box is good if we want both string or array

Reply:
Changed my mind, by default we'll support comma separated string. If there's a comma `dpm` query param becomes an array